test/jdk/javax/swing/JMenuItem/RightLeftOrientation.java

```diff
@@ -54,27 +72,48 @@
 import javax.swing.KeyStroke;
 import javax.swing.LookAndFeel;
 import javax.swing.SwingConstants;
```

Comment:
```suggestion
import javax.swing.KeyStroke;
import javax.swing.SwingConstants;
```
`LookAndFeel` is unused now.